book/animations.md

```diff
@@ -1974,8 +1974,10 @@ class Transform(VisualEffect):
 
 class ClipRRect(VisualEffect):
     def map(self, rect):
-        bounds = self.rrect.rect()
-        bounds.intersect(rect)
+        bounds = skia.Rect.MakeXYWH(
+            rect.x(), rect.y(),
+            rect.width(), rect.height())
+        bounds.intersect(self.rrect.rect())
```

Comment:
I think the first three lines here are just copying `rect`, right? We usually use `.makeOffset(0.0, 0.0)` for that.